src/components/core/geoChart/geoChartDataSource.ts

```diff
@@ -0,0 +1,82 @@
+// (C) 2019-2020 GoodData Corporation
+import invariant = require("invariant");
+import { Execution } from "@gooddata/typings";
+import { DEFAULT_LATITUDE, DEFAULT_LONGITUDE, DEFAULT_PUSHPIN_SIZE_VALUE } from "../../../constants/geoChart";
+import { stringToFloat } from "../../../helpers/utils";
+import { IGeoDataIndex } from "../../../interfaces/GeoChart";
+
+type IGeoDataSourceFeature = GeoJSON.Feature<GeoJSON.Geometry, GeoJSON.GeoJsonProperties>;
+export type IGeoDataSourceFeatures = IGeoDataSourceFeature[];
+
+function getLocation(locationValue: Execution.IResultHeaderItem): [number, number] {
+    if (locationValue && Execution.isAttributeHeaderItem(locationValue)) {
+        const latlng = locationValue.attributeHeaderItem.name;
+        const [latitude, longitude] = latlng.split(";").map(stringToFloat);
+        if (!longitude) {
```

Comment:
Because, if `latlng` is a valid location string (with `;` inside),`latitude` will always exist.
No need for validating `latitude`.
What do you think?